ktor-server/ktor-server-plugins/ktor-server-forwarded-header/jvmAndNix/src/io/ktor/server/plugins/forwardedheaders/ForwardedHeaders.kt

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.forwardedheaders
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.plugins.*
+import io.ktor.server.request.*
+import io.ktor.util.*
+
+/**
+ * Configuration options for [ForwardedHeaders] plugin. Allows for defining the order in which headers
+ * are forwarded.
+ */
+public class ForwardedHeadersConfig {
+    public var forwardedHeadersHandler: (MutableOriginConnectionPoint, List<ForwardedHeaderValue>) -> Unit =
```

Comment:
Consider avoiding having public API with `var` lambdas. It's hard to track compatibility and migrate them: adding or changing parameter starts being hard to migrate breaking change